	\begin{abstract}
	%		The abstract should appear at the top of the left-hand column of text, about
	%		0.5 inch (12 mm) below the title area and no more than 3.125 inches (80 mm) in
	%		length.  Leave a 0.5 inch (12 mm) space between the end of the abstract and the
	%		beginning of the main text.  The abstract should contain about 100 to 150
	%		words, and should be identical to the abstract text submitted electronically
	%		along with the paper cover sheet.  All manuscripts must be in English, printed
	%		in black ink.
	Structural variants (SVs) -- such as insertions, deletions, and duplications of an individual’s genome -- have been associated with genetic diseases and promotion of genetic diversity. Common approaches to detect SVs in an unknown genome require sequencing fragments of the genome, comparing them to a high-quality reference genome, and predicting SVs based on identified discordant fragments. However, inferring SVs from sequencing data has proven to be a challenging mathematical and computational problem because true SVs are rare and prone to low-coverage noise. We developed a computational method which seeks to improve existing SV detection methods in three ways: First, we implement an optimization approach consisting of a negative binomial log-likelihood objective function. Second, we use a block-coordinate descent approach to simultaneously predict if an SV is homozygous or heterozygous given genomic data of related individuals. Third, we model a biologically realistic scenario where variants in the child are either inherited –and therefore must be present in the parent—or novel. We present results on simulated data, which demonstrate improvements in predicting SVs and uncovering true SVs from false positives. 
\end{abstract}